packages/sit-onyx/src/components/OnyxModal/examples/DefaultExample.vue

```diff
@@ -9,7 +9,7 @@ const isOpen = ref(false);
   <div>
     <OnyxButton label="Show modal" @click="isOpen = true" />
 
-    <OnyxModalDialog label="Example modal dialog" :open="isOpen" @close="isOpen = false">
+    <OnyxModal v-model:open="isOpen" label="Example modal dialog" non-dismissible>
```

Comment:
```suggestion
    <OnyxModal v-model:open="isOpen" label="Example modal dialog">
```

Shouldn't we support dismissable for the default example? Otherwise users might think that it is not supported